src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,46 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService
+                = (MovieService) injector.getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall coolestHall = new CinemaHall();
+        coolestHall.setCapacity(250);
+        coolestHall.setDescription("3d plus biggest screen, wow");
+        CinemaHallService cinemaHallService
+                = (CinemaHallService) injector.getInstance((CinemaHallService.class));
+        cinemaHallService.add(coolestHall);
+        System.out.println(cinemaHallService.get(coolestHall.getId()));
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(coolestHall);
+        movieSession.setShowTime(LocalDateTime.now().plusDays(1L));
+
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
+
+        movieSessionService.add(movieSession);
+        movieSessionService.findAvailableSessions(movieSession.getId(),
```

Comment:
again, why are you passing id of movie session here? what does this method expect?